features/step_definitions/digital_summary_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^the customer (.*) a digital appointment summary$/) do |requested|
+  @appointment_summary = fixture(:populated_appointment_summary).tap do |as|
+    as.requested_digital = case requested
+                           when 'requested' then true
+                           when 'did not request' then false
+                           end
+  end
+end
+
+Then(/^we should know that the customer (.*) a digital version$/) do |requested|
+  requested_digital = case requested
+                      when 'requested' then true
+                      when 'did not request' then false
+                      end
+
+  appointment_summary = AppointmentSummary.first
+  expect(appointment_summary.public_send(:requested_digital)).to eq(requested_digital)
```

Comment:
Sending the `requested_digital` by reflection isn't necessary here since it's just a regular attribute with public visibility.